go/vt/schema/ddl_strategy_test.go

```diff
@@ -320,6 +321,20 @@ func TestParseDDLStrategy(t *testing.T) {
 			runtimeOptions:   "",
 			cutOverThreshold: 5 * time.Minute,
 		},
+		{
+			strategyVariable:  "vitess --force-cut-over-after=3m",
+			strategy:          DDLStrategyVitess,
+			options:           "--force-cut-over-after=3m",
+			runtimeOptions:    "",
+			forceCutOverAfter: 3 * time.Minute,
+		},
+		{
+			strategyVariable: "vitess --force-cut-over-after=r3m",
+			strategy:         DDLStrategyVitess,
+			options:          "--force-cut-over-after=3m",
```

Comment:
Is it supposed to be different (valid) here?

Reply:
Fixed (`options` removed as it's never tested upon error)